surfer/io.py

```diff
@@ -10,6 +10,8 @@
 import logging
 logger = logging.getLogger('surfer')
 
+from .utils import verbose
```

Comment:
flake error, module level import goes above calls. Move it up three or more lines